Makefile

```diff
@@ -1,28 +1,43 @@
-SHELL := $(shell echo $$SHELL)
```

Comment:
This took me ~1h to figure out, apparently it breaks linux `make` in github actions.

Reply:
BUT WHY

Reply:
idk, you can see the error i got here: https://github.com/melange-re/melange/actions/runs/4365427888/jobs/7634028656#step:6:16

Reply:
also interestingly, macos works fine 🤷

Reply:
I thought this was a nice hack. very sad

Reply:
Just curious, why were you using it for?